### PR TITLE
Improve the training code to support the non-distributed job and add …

### PR DESCRIPTION
**Pull request by halio-g**
…the dashboard access.

**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
--- YOUR PR SUMMARY GOES HERE ---
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [X] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
ERROR 2023-01-24T08:18:07.710267918Z [resource.labels.taskName: workerpool1-0] distributed.comm.core.CommClosedError: in <distributed.comm.tcp.TCPConnector object at 0x7f467c362670>: ConnectionRefusedError: [Errno 111] Connection refused
ERROR 2023-01-24T08:18:07.710246451Z [resource.labels.taskName: workerpool1-0] raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
ERROR 2023-01-24T08:18:07.710241133Z [resource.labels.taskName: workerpool1-0] File "/usr/local/lib/python3.8/dist-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
ERROR 2023-01-24T08:18:07.710234136Z [resource.labels.taskName: workerpool1-0] convert_stream_closed_error(self, e)
ERROR 2023-01-24T08:18:07.710225682Z [resource.labels.taskName: workerpool1-0] File "/usr/local/lib/python3.8/dist-packages/distributed/comm/tcp.py", line 461, in connect
ERROR 2023-01-24T08:18:07.710219279Z [resource.labels.taskName: workerpool1-0] return fut.result()
ERROR 2023-01-24T08:18:07.710212520Z [resource.labels.taskName: workerpool1-0] File "/usr/lib/python3.8/asyncio/tasks.py", line 494, in wait_for
ERROR 2023-01-24T08:18:07.710206532Z [resource.labels.taskName: workerpool1-0] comm = await asyncio.wait_for(
ERROR 2023-01-24T08:18:07.710200289Z [resource.labels.taskName: workerpool1-0] File "/usr/local/lib/python3.8/dist-packages/distributed/comm/core.py", line 291, in connect
ERROR 2023-01-24T08:18:07.710193905Z [resource.labels.taskName: workerpool1-0] Traceback (most recent call last):
ERROR 2023-01-24T08:18:07.710187771Z [resource.labels.taskName: workerpool1-0]
ERROR 2023-01-24T08:18:07.710181643Z [resource.labels.taskName: workerpool1-0] The above exception was the direct cause of the following exception:
ERROR 2023-01-24T08:18:07.710172534Z [resource.labels.taskName: workerpool1-0]
ERROR 2023-01-24T08:18:07.710134266Z [resource.labels.taskName: workerpool1-0] ConnectionRefusedError: [Errno 111] Connection refused
ERROR 2023-01-24T08:18:07.707676716Z [resource.labels.taskName: workerpool1-0] ModuleNotFoundError: No module named 'pythonjsonlogger'
ERROR 2023-01-24T08:18:07.707607628Z [resource.labels.taskName: workerpool1-0] Error in sitecustomize; set PYTHONVERBOSE for traceback:
ERROR 2023-01-24T08:18:03.166011099Z [resource.labels.taskName: workerpool0-0] OSError: Timed out trying to connect to tcp://cmle-training-workerpool0-b4dbcb58d5-0:3333 after 1200 s

File: notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb [hunk 1]
```diff
@@ -61,7 +61,7 @@
       "source": [
```
Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>Replace Vertex AI SDK and CustomContainerTrainingJob with "Vertex AI Training"</p><p> </p><p>For steps, drop the first 2 bullets (Configure, Create)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>consists of different types of irises. -&gt; predicts the iris species.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>combine 2 code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>Is this a TRAIN_IMAGE not a DEPLOY_IMAGE?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-8:latest" - I recommend define this as DEPLOY_IMAGE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (andrewferlitsch) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>You need to delete the job</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Comment (jackiejackjack) [inline notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb:61]: <p>Can you write a one sentence explanation on what the docker file is/what it does/why you need it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

**Comment by halio-g** [comment]
Thanks for the review Jack and Andrew. Jack said that the pull request looks fine. Andrew, could you please let me know if you have other comments on it.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**